src/routes/+page.svelte

```diff
@@ -89,7 +174,33 @@
 	{#each sections as section, index}
 		<!-- Allow focus jumping to section to avoid focus styles being applied to input on Safari link click -->
 		<!-- svelte-ignore a11y_no_noninteractive_tabindex -->
-		<section id={`section-${index}`} tabindex={-1} bind:this={section.el}>
+		<section id={`section-${index}`} tabindex={-1} bind:this={section.el} use:storeSection>
+			<div
+				style="height: 100vh;
+					width: 100%;
+					z-index: -2;
+					position: fixed;
+					top: 0;
+					left: 0;
+					display: flex;"
```

Comment:
nit: could we change this inline styling to a class? Same for lines 191 and 199. I typically like to keep most of the styling out of inline for readability/consistency, unless there's a dynamic variable that needs to be applied